include/config/config_graphics.h

```diff
@@ -66,6 +66,10 @@
 // Disable the fix to Koopa's unshelled model.
 #define KOOPA_KEEP_PINK_SHORTS
 
+// Copies the object graph node (model) to the star dance cutscene's object.
```

Comment:
This needs a better name and description. It took me a while to understand how this was different than normal (since keys use the key model), until I actually looked at the implementation and realized that it would work for custom star models. Maybe something along the lines of STAR_DANCE_USES_STARS_MODEL? Open to suggestions.